docs/content/dagster-cloud/managing-deployments/managing-multiple-projects-and-teams.mdx

```diff
@@ -0,0 +1,177 @@
+---
+title: Managing multiple projects & teams in Dagster Cloud
+description: "How to set up and manage multiple projects with Dagster Cloud Hybrid deployments."
+---
+
+# Managing multiple projects & teams in Dagster Cloud
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+Whether to manage complexity or to divide your work into areas of responsibility, you may want to isolate your code bases into multiple projects. See [Strategies for code separation](#strategies-for-code-separation) for more information.
+
+If you require isolation between your execution context, see [Strategies for execution context isolation](#strategies-for-execution-context-isolation).
```

Comment:
"context" is a little vague, consider
s/"execution context"/"code execution environments"